apps/web/components/v2/eventtype/EventTypeSingleLayout.tsx

```diff
@@ -51,6 +52,7 @@ function EventTypeSingleLayout({
   team,
   disableBorder,
   enabledAppsNumber,
+  activeAppsNumber,
```

Comment:
It looks like web3 apps are not being captured by these two props

![image](https://user-images.githubusercontent.com/65426560/190290481-7fa4261c-b570-4a3b-9ede-eb026182b15c.png)